tvtk/tvtk_base.py

```diff
@@ -471,15 +478,28 @@ def update_traits(self, obj=None, event=None):
         vtk.vtkObject.GlobalWarningDisplayOff()
 
         for name, getter in self._updateable_traits_:
+            if name == 'global_warning_display':
```

Comment:
This one is not originally my code, I just moved it out of the try block.  So I don't know.